### PR TITLE
ACC-544 graphics download message box

### PR DESCRIPTION
**Pull request by thurstontye**
Superceeds https://github.com/Financial-Times/n-syndication/pull/77

### CODE REVIEW

File: src/js/messages.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	} else if (item.canDownload === 0) {
+		message = item.lang !== 'en' ? MESSAGES.MSG_4250 : MESSAGES.MSG_4200;
+	} else if (item.canDownload === -1) {
+		message = MESSAGES.MSG_4100;
+	} else {
+		message = MESSAGES.MSG_2000;
+	}
+
+	return interpolate(message, item);
+}
```
Comment (rosario) [inline src/js/messages.js:46]: The same code is in config:
https://github.com/Financial-Times/n-syndication/blob/d0adca65350daab118fc0505c8743662e6f7de15/src/js/config.js#L33-L73

File: src/js/user-access.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import {products as getUserProducts} from 'next-session-client';
+import { SYNDICATION_ACCESS } from './config';
+export async function checkIfUserIsSyndicationCustomer () {
+
+	const response = await getUserProducts().catch(err => err);
```
Comment (rosario) [inline src/js/user-access.js:5]: The `catch` part is not necessary since we are returning the same error

File: src/js/user-access.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+	return response && response.products
+		? response.products.includes(SYNDICATION_ACCESS.STANDARD)
+		: false;
+}
+export async function getSyndicationAccess () {
+
+	const response = await getUserProducts().catch(err => err);
```
Comment (rosario) [inline src/js/user-access.js:13]: Similar to above

File: src/js/messages.js (hunk 1 above)
Comment (thurstontye) [reply src/js/messages.js:46]: Sorry that some bad copy pasta on my part. Decided ot move the messaging funtionality out of config into it's own thing and forgot to remove it from config. Update on it's way.

File: src/js/user-access.js (hunk 2 above)
Comment (thurstontye) [reply src/js/user-access.js]: done

File: src/js/user-access.js (hunk 3 above)
Comment (thurstontye) [reply src/js/user-access.js]: done

File: src/js/messages.js (hunk 1 above)
Comment (thurstontye) [reply src/js/messages.js:46]: Done. Removed from `config.js`

**Review by rosario — APPROVED** [review]

File: src/js/index.js [hunk 4]
```diff
@@ -1,27 +1,20 @@
...
+export async function init (flags) {
 	if (!flags.get('syndication')) {
 		return;
 	}
 
-	const userIsSyndicationCustomer = await checkIfUserIsSyndicationCustomer();
-	if (!userIsSyndicationCustomer) {
+	const syndicationAccess = await getSyndicationAccess();
+
+	if (syndicationAccess.length === -1 || !syndicationAccess.includes(SYNDICATION_ACCESS.STANDARD)) {
```
Comment (AniaMakes) [inline src/js/index.js:17]: Why the comparison to `-1`? As far as I can tell, we can get either an empty array or an array of stuff... Were you thinking of `findIndex`?

**Review by AniaMakes — COMMENTED** [review]
Overall this looks pretty solid. It's really hard to review PRs like this because one needs to scroll back and forth to reference, and if you download stuff locally then it's tricky to see the diff as well. 

Good work on tests! Hopefully we can deal with the lack of tests on a PR by PR basis to have a functional suite down the line!

File: src/js/index.js (hunk 4 above)
Comment (thurstontye) [reply src/js/index.js:17]: oops good spot.

**Pull request merged**